eng/pipelines/templates/stages/archetype-rust-release.yml

```
@@ -77,8 +77,8 @@ stages:
           timeoutInMinutes: 120
           dependsOn: TagRepository
           pool:
-            name: azsdk-pool-mms-ubuntu-2004-general
-            image: azsdk-pool-mms-ubuntu-2004-1espt
+            name: azsdk-pool
+            image: ubuntu-24.04
```

We should publish on the minimal supported OS just in case.
```suggestion
            image: ubuntu-22.04
```